src/main/java/com/actiontech/dble/config/model/SystemConfig.java

```diff
@@ -161,6 +161,11 @@
     private ProblemReporter problemReporter;
     private boolean useOuterHa = false;
 
+    private boolean enableFlowControl = true;
+    private int flowControlStartThreshold = 4096;
```

Comment:
the serverConifg param add need to be added in both:
+ server_temp.xml
+ ShowSysParam

Would you like to add them all?

Reply:
No problem.